pipeline.cpp

```diff
@@ -74,14 +74,66 @@ void DrawLine(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* cons
     // Your code goes here
 }
 
+int MaxNum(const int & num1, const int & num2)
```

Comment:
I'm glad to see I'm not the only one who failed to realize that there was already functions declared for us that took care of finding the max and the min of three numbers. (check under definitions.h).
Still, I like this really simple implementation. I was lazy and just imported the <cmath> library.

Reply:
@cynblkfox Hey, I actually did the same thing.